autopeering/peer/peer.go

```diff
@@ -126,3 +129,37 @@ func Unmarshal(data []byte) (*Peer, error) {
 	}
 	return FromProto(s)
 }
+
+type peerJSON struct {
+	PublicKey string          `json:"publicKey"`
+	IP        net.IP          `json:"ip"`
+	Services  *service.Record `json:"services"`
+}
+
+// UnmarshalJSON de-serializes given json data into a Peer struct.
```

Comment:
```suggestion
// UnmarshalJSON deserializes given json data into a Peer struct.
```

just for consistency as you wrote `deserializes` at `func (s *Record) UnmarshalJSON`

Reply:
I have no idea why I wrote it like that here :) Thanks for fixing!